### PR TITLE
Ce 1859 modal styling

### PR DESCRIPTION
**Pull request by kamilkoterba**
https://wikia-inc.atlassian.net/browse/CE-1859

Adding styles for flags edit form on modal.

@Wikia/community-engineering

### CODE REVIEW

File: extensions/wikia/Flags/Flags.i18n.php [hunk 1]
```diff
@@ -10,7 +10,7 @@
  */
 $messages['en'] = [
 	'flags-description' => 'Portable flags',
-	'flags-edit-form-more-info' => 'More info',
+	'flags-edit-form-more-info' => 'More info >',
```
Comment (adamkarminski) [inline extensions/wikia/Flags/Flags.i18n.php]: What about › ( &rsaquo; )?

File: extensions/wikia/Flags/controllers/templates/FlagsController_editForm.php [hunk 2]
```diff
@@ -8,13 +8,20 @@
...
+			$flagParamsNames = json_decode( $flag['flag_params_names'] );
+			if ( !empty( $flagParamsNames ) ): ?>
+				<fieldset class="params">
+				<?php
 				foreach ( $flagParamsNames as $flagParamName => $flagParamDescription ):
 					$flagParamValue = (string)$flag['params'][$flagParamName];
-			?>
-			<input type="text" name="<?= Sanitizer::encodeAttribute( "{$prefix}:{$flagParamName}" ) ?>" value="<?= Sanitizer::encodeAttribute( $flagParamValue ) ?>" placeholder="<?= Sanitizer::encodeAttribute( $flagParamDescription ) ?>" class="param">
-			<?php
+					?>
```
Comment (adamkarminski) [inline extensions/wikia/Flags/controllers/templates/FlagsController_editForm.php]: IMHO it's better to leave the `?>` tag in the same column that the opening tag. What do you think?

File: extensions/wikia/Flags/Flags.i18n.php (hunk 1 above)
Comment (kamilkoterba) [inline extensions/wikia/Flags/Flags.i18n.php]: &rsaquo; renders poorly - drops to the bottom instead of vertical middle. Leaving > for now.

Comment (adamkarminski) [inline extensions/wikia/Flags/Flags.i18n.php]: :+1:

**Comment by adamkarminski** [comment]
LGTM :+1:

**Comment by lukaszkonieczny** [comment]
lgtmaw ;)

**Pull request merged**